tools/crm_resource.c

```diff
@@ -946,18 +1071,19 @@ delete()
 }
 
 static int
-list_agents(const char *spec, crm_exit_t *exit_code)
+list_agents(crm_exit_t *exit_code)
```

Comment:
The idea with adding spec as a parameter here (and likewise in several other functions) was to make it more standalone in case it needed to get moved elsewhere as part of converting to formatted output.  I'm still undecided on if I'm going to do that or not.

Reply:
Good point, I'll add it back